main.go

```diff
@@ -24,74 +24,52 @@ import (
 	"google.golang.org/grpc"
 )
 
+const (
+	flagPrefixZerolog = "log"
+	flagPrefixMetrics = "metrics"
+	flagPrefixHTTP    = "http"
+)
+
 func main() {
 	rootCmd := &cobra.Command{
 		Use:   "prom-authzed-proxy",
 		Short: "Proxy that gates access to Prometheus based on an Authzed Check call",
 		PreRunE: cobrautil.CommandStack(
 			cobrautil.SyncViperPreRunE("prom-authzed-proxy"),
-			cobrautil.ZeroLogPreRunE,
+			cobrautil.ZeroLogPreRunE(flagPrefixZerolog, zerolog.InfoLevel),
 		),
 		Run: rootRun,
 	}
 
-	rootCmd.Flags().String("upstream-prom-addr", "", "address of the upstream Prometheus")
-	rootCmd.Flags().String("metrics-addr", ":9090", "address to listen on for the metrics server")
-	rootCmd.Flags().StringSlice("cors-allowed-origins", []string{"*"}, "allowed origins for CORS requests")
-
-	rootCmd.Flags().String("local-addr", ":80", "address to listen on for web requests")
-	rootCmd.Flags().String("local-key-path", "", "local path to the TLS key for the proxy server")
-	rootCmd.Flags().String("local-cert-path", "", "local path to the TLS certificate for the proxy server")
+	cobrautil.RegisterZeroLogFlags(rootCmd.Flags(), flagPrefixZerolog)
+	cobrautil.RegisterHttpServerFlags(rootCmd.Flags(), flagPrefixMetrics, "metrics", ":9090", true)
+	cobrautil.RegisterHttpServerFlags(rootCmd.Flags(), flagPrefixHTTP, "http", ":8443", true)
 
+	rootCmd.Flags().StringSlice(flagPrefixHTTP+"-cors-allowed-origins", []string{"*"}, "allowed origins for CORS requests")
+	rootCmd.Flags().String("upstream-prom-addr", "", "address of the upstream Prometheus")
 	rootCmd.Flags().String("object-id-parameter", "", "the name of the query parameter in incoming calls to use as the object ID in Authzed Check calls")
 
 	rootCmd.Flags().String("authzed-endpoint", "grpc.authzed.com:443", "address of the Authzed to use for checking")
 	rootCmd.Flags().String("authzed-tls-cert-path", "", "path at which to find a certificate for authzed TLS")
 	rootCmd.Flags().String("authzed-token", "", "authzed token to use for checking tenancy")
 	rootCmd.Flags().Bool("authzed-insecure", false, "connect to Authzed without TLS")
-
 	rootCmd.Flags().String("authzed-object-definition-path", "", "full object definition path in Authzed to check")
 	rootCmd.Flags().String("authzed-permission", "", "permission in Authzed to check")
-
 	rootCmd.Flags().String("authzed-subject-definition-path", "", "full subject definition path in Authzed to check")
-	rootCmd.Flags().String("authzed-subject-relation", "...", "subject relation in Authzed to check. Defaults to ...")
-
-	cobrautil.RegisterZeroLogFlags(rootCmd.Flags())
-
-	rootCmd.Execute()
-}
+	rootCmd.Flags().String("authzed-subject-relation", "", "subject relation in Authzed to check. Defaults to ...")
```

Comment:
This will need to change to empty for the V1 API

Reply:
i did all of those changes already?

Reply:
oh the description is lying now and still saying ellipsis!